static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -256,7 +263,7 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
 
   const onOpenLink = e => {
     e.stopPropagation();
-    const provider = match!.config?.provider;
+    const provider = match?.config?.provider;
```

Comment:
Maybe we could use `frame_source_link` as the provider when we get it from the event data?